runbotics/runbotics-scheduler/src/queue/process/process.controller.ts

```diff
@@ -14,16 +14,16 @@ export class ProcessController {
     constructor(private queueService: QueueService) { }
 
     @FeatureKeys(FeatureKey.PROCESS_START)
-    @Post(':processInfo/start')
+    @Post(':processId/start')
```

Comment:
take a look at validation schema `startProcessSchema` here it should be adjusted as well